src/app/modals/delete-agent-validation-modal/delete-agent-validation-modal.component.ts

```diff
@@ -0,0 +1,38 @@
+import { Component } from '@angular/core';
+import { AgentManagementComponent } from '../../agent-management/agent-management.component';
+import { NgbActiveModal, NgbModal } from '@ng-bootstrap/ng-bootstrap';
+import { ToastClassEnum } from '../../enums/toast-class-enum';
+import { ToastService } from '../../services/toast.service';
+import { AgentService } from '../../services/agent.service';
+import { Router } from '@angular/router';
+
+@Component({
+  selector: 'app-delete-agent-validation-modal',
+  templateUrl: './delete-agent-validation-modal.component.html',
+  styleUrl: './delete-agent-validation-modal.component.css'
+})
+export class DeleteAgentValidationModalComponent {
+  agentComponent: AgentManagementComponent
+  agentId: number
+
+  constructor(public activeModal: NgbActiveModal, private modalService: NgbModal, private apiService: AgentService, private toastService: ToastService, private route: Router) { }
```

Comment:
private modalService: NgbModal not being used

Reply:
done